executor/slow_query.go

```diff
@@ -490,6 +491,34 @@ func getLineIndex(offset offset, index int) int {
 	return fileLine
 }
 
+// splitByColon split a line like "field: value field: value..."
+func splitByColon(line string) (fields []string, values []string) {
+	re := regexp.MustCompile(`\w+: `)
+	matches := re.FindAllStringIndex(line, -1)
```

Comment:
Actually, I don't want to use regexp here, It will have a performance issue.

Reply:
I worried about it too. That's why the little format change is the first [solution](https://github.com/pingcap/tidb/pull/27950) I choosed, and I didn't close it. Any other good idea would be appreciated.